core-bundle/src/Resources/contao/library/Contao/Environment.php

```diff
@@ -290,15 +290,15 @@ protected static function httpUserAgent()
 	 */
 	protected static function httpHost()
 	{
-		if (!empty($_SERVER['HTTP_HOST']))
+		if (isset($_SERVER['HTTP_HOST']) && !empty($_SERVER['HTTP_HOST']))
```

Comment:
`empty()` already incoroporates `isset`, so this is not necessary.

> No warning is generated if the variable does not exist. That means empty() is essentially the concise equivalent to `!isset($var) || $var == false`. 

https://www.php.net/manual/en/function.empty.php

Reply:
Thanks! I should not do that in the middle of the night :-D